sphere-models/src/main/java/io/sphere/sdk/products/attributes/LocalizedToStringProductAttributeConverter.java

```diff
@@ -0,0 +1,231 @@
+        return convertReference(channelReference, productType);
+    }
+
+    @Override
+    protected String convertCategoryReferenceSet(final Set<Reference<Category>> categoryReferenceSet, final Attribute attribute, final ProductType productType) {
+        return convertReferenceSet(categoryReferenceSet, productType);
+    }
+
+    @Override
+    protected String convertCategoryReference(final Reference<Category> categoryReference, final Attribute attribute, final ProductType productType) {
+        return convertReference(categoryReference, productType);
+    }
+
+    @Override
+    protected String convertBooleanSet(final Set<Boolean> booleanSet, final Attribute attribute, final ProductType productType) {
+        return collectToString(booleanSet.stream().sorted().map(value -> convertBoolean(value, attribute, productType)));
+    }
+
+    @Override
+    protected String convertString(final String stringValue, final Attribute attribute, final ProductType productType) {
+        return stringValue;
+    }
+
+    @Override
+    protected String convertMoney(final MonetaryAmount moneyValue, final Attribute attribute, final ProductType productType) {
+        final MonetaryAmountFormat monetaryAmountFormat = MonetaryFormats.getAmountFormat(locale());
+        return monetaryAmountFormat.format(moneyValue);
+    }
+
+    @Override
+    protected String convertLocalizedString(final LocalizedString localizedString, final Attribute attribute, final ProductType productType) {
+        return localizedString.getTranslation(getLocales());
+    }
+
+    @Override
+    protected String convertLocalizedEnumValue(final LocalizedEnumValue localizedEnumValue, final Attribute attribute, final ProductType productType) {
+        return localizedEnumValue.getLabel().getTranslation(getLocales());
+    }
+
+    @Override
+    protected String convertEnumValue(final EnumValue enumValue, final Attribute attribute, final ProductType productType) {
+        return enumValue.getLabel();
+    }
+
+    @Override
+    protected String convertDateTime(final ZonedDateTime dateTimeValue, final Attribute attribute, final ProductType productType) {
+        final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT).withLocale(locale());
+        return dateTimeFormatter.format(dateTimeValue);
+    }
+
+    @Override
+    protected String convertDate(final LocalDate dateValue, final Attribute attribute, final ProductType productType) {
+        final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM).withLocale(locale());
+        return dateTimeFormatter.format(dateValue);
+    }
+
+    @Override
+    protected String convertBoolean(final Boolean booleanValue, final Attribute attribute, final ProductType productType) {
+        return booleanValue.toString();
+    }
+}
```

Comment:
I think date is a good example:
the dates: 15.10. and 20.09
sorted as Date: 20.09., 15.10.
sorted as String: 15.10., 20.09.

So for dates it is better to sort the values first and then use `toString()`.

For localized Strings it is better to sort the translated Strings rather than the objects containing all information.